zerver/views/invite.py

```diff
@@ -48,6 +48,9 @@ def invite_users_backend(
     request: HttpRequest,
     user_profile: UserProfile,
     invitee_emails_raw: str = REQ("invitee_emails"),
+    notify_referrer_on_join: bool = REQ(
+        "notify_referrer_on_join", json_validator=check_bool, default=True
+    ),
```

Comment:
These REQ parameters are sorted; please preserve that, and take care with similar sorted elements elsewhere in this file.